_includes/v21.1/orchestration/kubernetes-expand-disk-helm.md

```diff
@@ -97,88 +76,43 @@ You can expand certain [types of persistent volumes](https://kubernetes.io/docs/
     {{site.data.alerts.end}}
 
-5. Examine the persistent volume claim. If the volume has a file system, you will see a `FileSystemResizePending` condition with an accompanying message:
+1. Examine the persistent volume claim. If the volume has a file system, you will see a `FileSystemResizePending` condition with an accompanying message:
 
-	<section class="filter-content" markdown="1" data-scope="helm">
 	{% include copy-clipboard.html %}
     ~~~ shell
     $ kubectl describe pvc datadir-my-release-cockroachdb-0
     ~~~
-	</section>
-
-	<section class="filter-content" markdown="1" data-scope="manual">
-	{% include copy-clipboard.html %}
-    ~~~ shell
-    $ kubectl describe pvc datadir-cockroachdb-0
-    ~~~
-	</section>
 
     ~~~
     Waiting for user to (re-)start a pod to finish file system resize of volume on node.
     ~~~
 
-6.  Delete the corresponding pod to restart it:
+1.  Delete the corresponding pod to restart it:
 
-	<section class="filter-content" markdown="1" data-scope="helm">
 	{% include copy-clipboard.html %}
     ~~~ shell
     $ kubectl delete pod my-release-cockroachdb-0
     ~~~
-   	</section>
-
-	<section class="filter-content" markdown="1" data-scope="manual">
-	{% include copy-clipboard.html %}
-    ~~~ shell
-    $ kubectl delete pod cockroachdb-0
-    ~~~
-	</section>
 
     The `FileSystemResizePending` condition and message will be removed.
 
-7. View the updated persistent volume claim:
+1. View the updated persistent volume claim:
 
-	<section class="filter-content" markdown="1" data-scope="helm">
 	{% include copy-clipboard.html %}
     ~~~ shell
     $ kubectl get pvc datadir-my-release-cockroachdb-0
     ~~~
 
     ~~~
 	NAME                               STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
-datadir-my-release-cockroachdb-0   Bound    pvc-75dadd4c-01a1-11ea-b065-42010a8e00cb   200Gi      RWO            standard       20m
+    datadir-my-release-cockroachdb-0   Bound    pvc-75dadd4c-01a1-11ea-b065-42010a8e00cb   200Gi      RWO            standard       20m
     ~~~	
-	</section>
-
-	<section class="filter-content" markdown="1" data-scope="manual">
-	{% include copy-clipboard.html %}
-    ~~~ shell
-    $ kubectl get pvc datadir-cockroachdb-0
-    ~~~
-
-    ~~~
-	NAME                    STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
-datadir-cockroachdb-0   Bound    pvc-75dadd4c-01a1-11ea-b065-42010a8e00cb   200Gi      RWO            standard       20m
-    ~~~
-	</section>
 
-8. The CockroachDB cluster needs to be expanded one node at a time. Repeat steps 3 - 6 to increase the capacities of the remaining volumes by the same amount.
\ No newline at end of file
+1. The CockroachDB cluster needs to be expanded one node at a time. Repeat steps 3 - 6 to increase the capacities of the remaining volumes by the same amount.
```

Comment:
I'm guessing these are auto-generated now

Reply:
Only if the customer is using a storage class.  If they statically provisioned the PVs this is still correct.